IdentityCore/src/webview/embeddedWebview/MSIDAADOAuthEmbeddedWebviewController.m

```diff
@@ -82,6 +82,7 @@ - (BOOL)decidePolicyAADForNavigationAction:(WKNavigationAction *)navigationActio
         decisionHandler(WKNavigationActionPolicyCancel);
         [MSIDPKeyAuthHandler handleChallenge:requestURL.absoluteString
                                      context:self.context
+                               customHeaders:[self customHeaders]
```

Comment:
nit: You're using a property here, right? Not calling a method? In that case you can use the self.customHeaders notation.